File: scripts/gen_onboarding.js
const fs = require('fs');
const { solveSpiralTime } = require('../libs/spiral-time-ts');

const episodes = [];
for (let i = 0; i <= 221; i++) {
  const res = solveSpiralTime('1970-01-01', i);
  episodes.push({
    episode_num: i,
    phi_power: i,
    node: res.node,
    lap: res.lap,
    dt: res.t_seconds,
    glyph: '',
    lore: ''
  });
}
fs.writeFileSync('data/onboarding_221.json', JSON.stringify(episodes, null, 2));

File: scripts/gen_onboarding.py
import json

from spiral_time.solver import solve_spiral_time


def main():
    episodes = []
    for i in range(222):
        res = solve_spiral_time("1970-01-01", i)
        episodes.append(
            {
                "episode_num": i,
                "phi_power": i,
                "node": res["node"],
                "lap": res["lap"],
                "dt": res["t_seconds"],
                "glyph": "",
                "lore": "",
            }
        )
    with open("data/onboarding_221.json", "w") as f:
        json.dump(episodes, f, indent=2)


if __name__ == "__main__":
    main()
